src/app.module.ts

```diff
@@ -21,12 +24,12 @@ import {PrismaModule} from './prisma/prisma.module';
         transport:
           process.env.NODE_ENV !== 'production'
```

Comment:
Let's use constants forthe hard-coded strings here.

Reply:
+1 Better to create a enum say `ENVIRONMENT` with `production`, `dev`, `qa` etc. fields